src/melee.cpp

```diff
@@ -1204,14 +1206,20 @@ bool player::can_weapon_block() const
 {
     return (weapon.has_technique( WBLOCK_1 ) ||
             weapon.has_technique( WBLOCK_2 ) ||
-            weapon.has_technique( WBLOCK_3 ));
+            weapon.has_technique( WBLOCK_3 ) ||
+            weapon.has_technique( WBLOCK_4 ) ||
+            weapon.has_technique( WBLOCK_5 ));
```

Comment:
This is already getting ugly.
Can be replaced with `blocking_ability( weapon ) > 0`.